monai/handlers/checkpoint_saver.py

```diff
@@ -37,6 +37,9 @@ class CheckpointSaver:
         name: identifier of logging.logger to use, if None, defaulting to ``engine.logger``.
         file_prefix: prefix for the filenames to which objects will be saved.
         save_final: whether to save checkpoint or session at final iteration or exception.
+            if want to save checkpoint when exception happened, please put this handler before
+            `StatsHandler` in the handler list, because the logic with ignite can only trigger
```

Comment:
"ignite" -> "Ignite", and the same elsewhere in the changes.

Reply:
updated in the latest commit.
Thanks.